### PR TITLE
Minor modifications

### PR DESCRIPTION
**Pull request by CodeBojan**
Added a mechanism to detect whether the CAN BUS is receiving messages or not. If it is not receiving messages, the proper console message will be displayed. Added shutdown script and modified exe script slightly.

### CODE REVIEW

File: src/can_service.py [hunk 1]
```diff
@@ -141,7 +141,10 @@ def read_can(execution_flag, config_flag, init_flags, can_lock):
     customLogger.debug("CAN process started!")
 
     period = 2
+    period_counter = 0
```
Comment (sale3) [inline src/can_service.py:144]: Add a short description of the variable in a comment.

File: src/can_service.py [hunk 2]
```diff
@@ -141,7 +141,10 @@ def read_can(execution_flag, config_flag, init_flags, can_lock):
...
+    previous_message_counter = 0
```
Comment (sale3) [inline src/can_service.py:145]: Add a short description of the variable in a comment.

File: src/can_service.py [hunk 3]
```diff
@@ -549,6 +559,8 @@ def __init__(self, temp_client, load_client, fuel_client):
             fuel_client.connect()
         self.fuel_client = fuel_client
 
+        self.message_counter = 0
```
Comment (sale3) [inline src/can_service.py:562]: Add a short description of the variable in a comment.

**Review by sale3 — CHANGES_REQUESTED** [review]

File: src/signal_control.py [hunk 4]
```diff
@@ -49,7 +49,8 @@ def __init__(self, sigs, flags):
         self.flags = flags
         self.original_handlers = []
         for sig in self.sigs:
-            self.original_handlers.append(signal.getsignal(sig))
+            if sig > signal.CTRL_BREAK_EVENT:
```
[the remark this answers is not in the document]
Comment (stekap000) [reply src/signal_control.py:52]: Given if condition should not be there at all.

File: src/can_service.py [hunk 5]
```diff
@@ -141,8 +141,11 @@ def read_can(execution_flag, config_flag, init_flags, can_lock):
     customLogger.debug("CAN process started!")
 
     period = 2
-    period_counter = 0
-    previous_message_counter = 0
+    period_counter = 0  # if the counter reaches 5, 10 seconds have passed, and then the check is made whether the bus
```
Comment (sale3) [inline src/can_service.py:144]: Looks good, just move the comments above the variable. Same goes for other variables.

**Review by sale3 — CHANGES_REQUESTED** [review]

**Review by stekap000 — APPROVED** [review]

**Review by sale3 — APPROVED** [review]

**Review by stekap000 — APPROVED** [review]

**Pull request merged**